linux/ibus-keyman/debian/watch

```diff
@@ -0,0 +1,2 @@
+version=4
+https://downloads.keyman.com/linux/alpha/10\.99\.(.+)/ibus-keyman-(.+)\.tar\.gz debian uupdate
```

Comment:
We'll need to remember to update this URL when we go to beta / stable.

Reply:
> 
> 
> We'll need to remember to update this URL when we go to beta / stable.

urls update to handle alpha, beta and stable and get the latest from any of them